Makefile

```diff
@@ -52,12 +52,17 @@ build/llvm.BUILT:
 	touch build/llvm.BUILT
 
 build/wasi-libc.BUILT: build/llvm.BUILT
-	$(MAKE) -C $(ROOT_DIR)/src/wasi-libc \
-		WASM_CC=$(PREFIX)/bin/clang \
-		SYSROOT=$(PREFIX)/share/wasi-sysroot
+	cmake -G Ninja \
+		--install \
+		-DCMAKE_SYSTEM_NAME=Generic \
+		-DCMAKE_AR=$(ROOT_DIR)/bin/llvm-ar \
+		-DCMAKE_NM=$(ROOT_DIR)/bin/llvm-nm \
```

Comment:
This is pre-existing, but we can remove the "llvm-" from these commands, as we install a plain "nm" and "ar" now.